tardis/adapters/sites/cloudstack.py

```diff
@@ -60,23 +61,11 @@ def __init__(self, machine_type: str, site_name: str):
     ) -> AttributeDict:
         response = await self.cloud_stack_client.deployVirtualMachine(
             name=resource_attributes.drone_uuid,
-            **self.configuration.MachineTypeConfiguration[self._machine_type],
+            **self._configuration.MachineTypeConfiguration[self._machine_type],
```

Comment:
What is the difference here to the newly introduced ``_machine_meta_data``. Why do you use the ``_configuration`` for access here?

Reply:
Good catch! I will fix that. There is no difference.

Reply:
If implementations rely on this attribute, yes. You should decide if the *MachineTypeConfiguration* is similar to e.g. *MachineMetaData*. If it is, it should be handled similarly.

Reply:
I have implemented a property to access the `machine_type_configuration`.